src/internal/restgateway/restgateway.go

```diff
@@ -0,0 +1,114 @@
+package restgateway
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/pachyderm/pachyderm/v2/src/internal/client"
+	"github.com/pachyderm/pachyderm/v2/src/internal/log"
+
+	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
+
+	pfsgw "github.com/pachyderm/pachyderm/v2/src/pfs"
```

Comment:
I think it's relatively atypical to import the protos with an alias.  In some places it's necessary, but it's not in this package.  I'd just use these as `pfs.RegisterAPIHandler` and so on.

Reply:
done